packages/waku/src/router/create-pages.ts

```diff
@@ -278,7 +282,7 @@ export const createPages = <
     {
       render: 'static' | 'dynamic';
       pathSpec: PathSpec;
-      handlers: Partial<Record<Method, ApiHandler>>;
+      handlers: Partial<Record<Method | 'all', ApiHandler>>;
```

Comment:
I don't dislike `'all'`. But, just another idea is `'*'`.

Reply:
hm it would seem weird to me to see if the object was called literally, like:

```ts
createApi({
  render: 'dynamic',
  handlers: {
    '*': handler,
    GET: getHandler
  }
})
```

What about using the name `default`? At first I thought it may be weird, but I think it might be intuitive?

Reply:
"all" sounds better to me than "default". I don't have a strong opinion though.

Reply:
I'll leave it as is then?

Reply:
👍